src/linear_algebra/basic.lean

```diff
@@ -740,11 +740,19 @@ le_antisymm (span_le.2 h₁) h₂
 @[simp] lemma span_eq : span R (p : set M) = p :=
 span_eq_of_le _ (subset.refl _) subset_span
 
-lemma map_span (f : M →ₗ[R] M₂) (s : set M) :
+lemma _root_.linear_map.map_span (f : M →ₗ[R] M₂) (s : set M) :
```

Comment:
It seems a little odd to me to have lemmas about `submodule.map` not start with `submodule.map_...`, but I understand that this way makes dot notation marginally more useful, so it's probably reasonable.

Assuming other reviewers agree with you, can you rename `span_preimage_le` below too, which is essentially the same pattern as `map_span_le`?

Reply:
I suppose another option is to just use `alias` or maybe `export` to enable dot notation while also keeping a copy of this lemma in an easy place to find.

Reply:
I don't have strong feelings here, but I tried the alias idea. Let's see if CI likes it.